src/zenml/integrations/constants.py

```diff
@@ -21,6 +21,7 @@
 FACETS = "facets"
 FEAST = "feast"
 GCP = "gcp"
+GCPSecretsManager = "gcp_secrets_manager"
```

Comment:
I think constants are usually all upper case and with underscores, e.g. `GCP_SECRETS_MANAGER`